t/05-distribution.t

```diff
@@ -8,14 +8,14 @@ use Mojo::Base -strict, -signatures;
 use FindBin '$Bin';
 use lib "$Bin/../external/os-autoinst-common/lib";
 use OpenQA::Test::TimeLimit '5';
+use distribution;
```

Comment:
I would move this line below the other `use` lines with the unit test related modules.